lib/Basics/tri-zip.cpp

```diff
@@ -419,7 +419,7 @@ int TRI_Adler32(char const* filename, uint32_t& checksum) {
     return TRI_set_errno(TRI_ERROR_SYS_ERROR);
   }
 
-  ssize_t chunkRemain = static_cast<TRI_read_t>(statbuf.st_size);
+  TRI_read_t chunkRemain = static_cast<TRI_read_reply_t>(statbuf.st_size);
```

Comment:
this makes no sense, as we are first casting to `TRI_read_reply_t` and then implicitly cast to `TRI_read_t` :thinking: